pkg/alluxiocluster/create_alluxiocluster.go

```diff
@@ -77,3 +78,9 @@ func CreateAlluxioClusterIfNotExist(ctx AlluxioClusterReconcileReqCtx) error {
 	}
 	return nil
 }
+
+func tweakAlluxioClusterConfig(ctx AlluxioClusterReconcileReqCtx) {
+	if ctx.Dataset.Spec.MultiMount != nil && *ctx.Dataset.Spec.MultiMount {
+		ctx.AlluxioCluster.Spec.Properties["alluxio.mount.table.source"] = "ETCD"
```

Comment:
so this means if we have set multipleMountPoints config.yaml file to be true, then the k8s operator automatically adss this to alluxiocluster spec is it? if yes, can u name the functions to be injectAlluxioClusterConfig() and add some document on the function (does go add func javadoc on top of the func like in java do?) to say inject or modify necessary alluxio site properties based on different deployment flavor defined in DataSet